Doc/library/inspect.rst

```diff
@@ -722,18 +722,20 @@ function.
    ``from __future__ import annotations`` was used), :func:`signature` will
    attempt to automatically un-stringize the annotations using
    :func:`annotationlib.get_annotations`.  The
-   *globals*, *locals*, and *eval_str* parameters are passed
+   *globals*, *locals*, *eval_str*, and *annotation_format* parameters are passed
```

Comment:
It's a minor thing, but: `globals`, `locals`, and `eval_str` are passed in to parameters with the same name when calling `get_annotations`.  But `annotation_format` is passed in to a parameter with a different name (just `format`).  It might be nice to explicitly tell that to the user.

I realize the documentation doesn't explicitly detail where these parameters go, so this would be all-new text.  And when I try it in my head it always comes across as clumsy.  So... should we even bother to try?  Would anybody be confused if we didn't bother to explain it?

Reply:
Good point, I will add a separate sentence about the `annotation_format` parameter.